### DISCUSSION TITLE
Tell me your favorite songs!!!

### DISCUSSION THREAD

**Discussion by cielcs** [NAIST-SE/AttractingContributors#1026] [2024 Attraction Contributions]
https://github.com/cielcs/FavoriteSongs

Please share with me your favorite songs!
Actually, I rarely listen to music, so please give me some recommendations!

**Comment by kanariari3** [comment]
Amaging Grace

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Reply by cielcs** [reply]
Sure!
